### ISSUE TITLE
XDG_RUNTIME_DIR isn't set causing systemd user dbus service to fail..

### ISSUE THREAD

**Issue by cswl** [cylgom/ly#90]
Error: 

```
Failed to get properties: Process org.freedesktop.systemd1 exited with status 1

```

Relevant Source file.. :  https://github.com/cylgom/ly/blob/dde8de60bd155b0101f69ec2d3a9ac76dcb72e00/src/login.c#L139

Refrence: https://serverfault.com/questions/887283/systemctl-user-process-org-freedesktop-systemd1-exited-with-status-1

**Comment by qyzk** [comment]
I'm having same issue. My workaround is to set `XDG_RUNTIME_DIR` and `DBUS_SESSION_BUS_ADDRESS` (and other XDG variables) for both login shell and systemd user instance.

e. g.
```
# '/etc/profile' and '~/.config/environment.d/env.conf'
XDG_RUNTIME_DIR="/run/user/1000"
DBUS_SESSION_BUS_ADDRESS="unix:path=/run/user/1000/bus"
```

Reboot and confirm they are set with `printenv` and `systemctl --user show-environment`.

Other display managers set these variables automatically, and without these variables some applications may not work. So I think Ly should set them.

reference: https://bbs.archlinux.org/viewtopic.php?pid=1833997

**Comment by SirWindfield** [comment]
I us the tried it out. Both commands output both env variables but I still fail to start xinitrc. I remember that it worked around a year ago with i3 before i switched over to awesomewm

**Comment by qub1750ul** [comment]
Maybe worth trying this
```
environment="DISPLAY XAUTHORITY XDG_RUNTIME_DIR DBUS_SESSION_BUS_ADDRESS"

systemctl --user import-environment $environment
if command -v dbus-update-activation-environment > /dev/null 2>&1 ; then
         dbus-update-activation-environment $environment
fi
```

**Comment by cylgom** [comment]
Fixed in 1e6ec95, thank you for the detailed explanation.
